### DISCUSSION TITLE
(on screen) thumbnail size

### DISCUSSION THREAD

**Discussion by nicnet123** [photoprism/photoprism#3302] [General]
Hi - I don't want necessarily to change the underlying thumbnail size in storage, but I would like to fit more on screen. How can this be adjusted?
Coming from google photos, now have about 200k photos in photoprism - it's handling it fine, but with google photos, you could change the size of onscreen thumbnails with just ctrl + and -. Can't do the same on photoprism ...

**Comment by alexisLefebvre** [comment]
Did you try the option to change the number of photos?

> ![image](https://user-images.githubusercontent.com/2071331/228809462-874aeede-5dd5-491a-ba51-e0035826a17d.png)

You have to click several times to switch the display.

You can test on the demo: https://demo.photoprism.app/library/browse

**Reply by nicnet123** [reply]
It cycles between 3 views.
The max "thumbnails" I see on my monitor in those 3 views is 12 (across), and 7 (down), so a total of 84 images.
On google photos, I like to shrink them down to 41 across by 25 or so, about 1000 images in one screen.

**Reply by alexisLefebvre** [reply]
Ok I see your point. Zooming out on Google Photos make images smaller, while on PhotoPrism the size of the photos is constant, so we can't see more photos.

**Reply by lastzero** [reply]
Dynamic thumb sizes add complexity and are likely to introduce bugs in combination with virtual scrolling. So implementing this cannot be done quickly, especially since we are also working on a long list of other features right now. Also note that browsers have a request limit per domain and home users typically don't have a CDN to compensate for this, so loading 1000 images at once typically isn't a great idea without professional content delivery infrastructure.

**Reply by nicnet123** [reply]
> especially since we are also working on a long list of other features right now. 

Fair enough.
